_posts/2023-06-06-may-2023-roundup.md

```diff
@@ -0,0 +1,105 @@
+An interesting look at some vulnerabilities in the CAN bus in Toyata's newer vehicles. - Blake
+
+- [**pts.blog: How to make smaller C and C++ binaries**](https://ptspts.blogspot.com/2013/12/how-to-make-smaller-c-and-c-binaries.html)<br>
+Compiler best practices for how to build smaller binaries. - Tyler
+
+- [**Size Optimizations - Pigweed**](https://pigweed.dev/docs/size_optimizations.html)<br>
+A post from Pigweed at Google on producing smaller binaries while using C++. I really like the callout of when to use LTO: "Disable LTO unless absolutely necessary" and "When enabling LTO, carefully and thoroughly test the resulting binary". Couldn't agree more. - Tyler
+
+- [**What Happened To Execute-in-Place?**](https://semiengineering.com/what-happened-to-execute-in-place/)<br>
+A brief history and overview of XIP, how the definition has changed, and how trends in silicon fabrication have affected it. - Eric
+
+- [**Getting started with Thingy:53 and Zephyr - DEV Community**](https://dev.to/denladeside/getting-started-with-thingy53-and-zephyr-4m88)<br>
+A short guide on how to use Zephyr to develop for the Nordic Thingy:53. - Mariya
+
+- [**Hand Me Your SECRET, MCU! Microarchitectural Timing Attacks on Microcontrollers are Practical - Black Hat Asia 2023 | Briefings Schedule**](https://www.blackhat.com/asia-23/briefings/schedule/index.html#hand-me-your-secret-mcu-microarchitectural-timing-attacks-on-microcontrollers-are-practical-30579)<br>
+Side channel attack against TrustZone in ARMv8-m. - Noah
+
+- [**Makefile Tutorial By Example**](https://makefiletutorial.com/#top)<br>
+A tutorial/guide/cookbook for Make in an easily-digestible format compared to searching through the documentation like I do every time I have to refresh my brain. -Eric
+
+- [**Google Online Security Blog: $22k awarded to SBFT ‘23 fuzzing competition winners**](https://security.googleblog.com/2023/05/22k-awarded-to-sbft-23-fuzzing.html)<br>
+Google runs a fuzzing competition, results for this year! - Noah
+
+- [**Embedded Skills Tree — Embedded**](https://embedded.fm/blog/2023/5/19/embedded-skills-tree)<br>
+A fun skills tree! Color in the boxes and level up your skills. 
+
+- [**wrench embedded interpreter**](https://www.northarc.com/wrench/www/)<br>
+Interesting embedded scripting language. - François
+
+- [**Memory Allocation**](https://samwho.dev/memory-allocation/)<br>
+Great, in-depth article on memory allocation (malloc and free). - François
+
+- [**What is the value of using a proprietary compiler instead of using GCC? : embedded**](https://old.reddit.com/r/embedded/comments/13ozrn9/what_is_the_value_of_using_a_proprietary_compiler/)<br>
+A fun Reddit discussion on proprietary compilers vs GCC. - Tyler
+
+- [**Booting J-Link as CMSIS-DAP Debug Probe | MCU on Eclipse**](https://mcuoneclipse.com/2023/04/30/booting-j-link-as-cmsis-dap-debug-probe/)<br>
+Turns out you can use a jlink as a CMSIS-DAP compatible probe :thinking_face:. - Noah
```

Comment:
As much as it will pain Noah I think we need to remove the thinking_face emoji. It does not render :(